routers/web/org/teams.go

```diff
@@ -66,13 +66,47 @@ func TeamsAction(ctx *context.Context) {
 		err = ctx.Org.Team.AddMember(ctx.User.ID)
 	case "leave":
 		err = ctx.Org.Team.RemoveMember(ctx.User.ID)
+		if err != nil {
+			if models.IsErrLastOrgOwner(err) {
+				ctx.Flash.Error(ctx.Tr("form.last_org_owner"))
+			} else {
+				log.Error("Action(%s): %v", ctx.Params(":action"), err)
```

Comment:
```suggestion
				log.Warn("Action(%s): %v", ctx.Params(":action"), err)
```

Reply:
The rest of this file uses log.Error for these

Reply:
we may wana change it too ... since this is not gitea error but a user issue ..?

Reply:
nope it's not - it's an error from RemoveMember which is likely a DB error.